lib/rest2.js

```diff
@@ -1630,6 +1630,26 @@ class RESTv2 {
     return this._makeAuthRequest('/auth/w/pulse/add', params, cb, PulseMessage)
   }
 
+  /**
+   * @param {object} params             - parameters
+   * @param {string} params.parent      - the parent id of the Pulse message
+   * @param {string} params.title       - the title of your Pulse message
+   * @param {string} params.content     - content of your Pulse message
+   * @param {number} params.isPublic    - make Pulse message public
+   * @param {string} params.isPin       - make Pulse message pinned
+   * @param {string} params.attachments - Base64 encoded list of strings
+   * @param {Function} [cb]             - callback
+   * @returns {Promise} p
+   * @see https://docs.bitfinex.com/reference#rest-auth-pulse-add
+   */
+  addPulseComment (params, cb) {
+    const { parent = '' } = params || {}
+    if (_isEmpty(parent)) {
+      return cb(new Error('parent (pulse id value) is required for comments'))
```

Comment:
hey @avsek477 , could you use `this._cb` instead of direct cb here please? We should support both promise and callback approaches in pub lib :)

Reply:
sure.. thanks for pointing it out to me